src/LoginSignup/RegistrationAPI.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+export async function registerEmail(userEmail) {
+	if (typeof userEmail !== 'string') console.warn('userEmail parameter is not a string.');
+	await axios
```

Comment:
let result = await fn() 

why .then if you are going to async await, if you expect error, wrap it in a try/catch